test/DriverTest.php

```diff
@@ -349,4 +453,44 @@ private function getPermissionsFromStat(array $stat): string
     {
         return \substr(\decoct($stat["mode"]), 1);
     }
+
+    public function testChmod(): \Generator
+    {
+        $fixtureDir = Fixture::path();
+
+        $path = "{$fixtureDir}/small.txt";
+        $this->assertNotSame('0777', \substr(\sprintf('%o', \fileperms($path)), -4));
+        $this->assertNull(yield File\chmod($path, 0777));
+        \clearstatcache();
```

Comment:
Shouldn't `chmod` do the `clearstatcache()`?

Reply:
It was there for `fileperms()`  to work correctly. But maybe I can use `Amp\File\stat` to get them instead... I'll try.